Makefile

```
@@ -144,7 +144,7 @@ build-linux:
 	GOOS=linux GOARCH=$(if $(findstring aarch64,$(shell uname -m)) || $(findstring arm64,$(shell uname -m)),arm64,amd64) LEDGER_ENABLED=false $(MAKE) build
 
 $(BUILD_TARGETS): go.sum $(BUILDDIR)/
-	CGO_CFLAGS="-O -D__BLST_PORTABLE__" go $@ -mod=readonly $(BUILD_FLAGS) $(BUILD_ARGS) ./...
+	go $@ -mod=readonly $(BUILD_FLAGS) $(BUILD_ARGS) ./...
```

From my side

```suggestion
	CGO_CFLAGS="-D__BLST_PORTABLE__" go $@ -mod=readonly $(BUILD_FLAGS) $(BUILD_ARGS) ./...
```

already works so probably let's make the minimal change?

---

The blst README (https://github.com/supranational/blst/tree/master/bindings/go#build)
kind of advises against it, that's why I didn't do that.

In fact they are now suggesting `-O2`, will give that a try.

---

agreed with removing the optimisation, given that it might break things in the binding. It's possible that with O2 level optimisation our bug is gone but there is newly introduced ones that we haven't met yet